lib/Widgets/IncomingTransmissionListEntry.vala

```diff
@@ -40,7 +40,9 @@ public class Drop.Widgets.IncomingTransmissionListEntry : TransmissionListEntry
      * @param transmission Transmission to build the entry on.
      */
     public IncomingTransmissionListEntry (IncomingTransmission transmission) {
-        Object (transmission: transmission);
+        base ();
```

Comment:
Yes, that's what caused the parent's constructor to not run before, now im just forcing it to run